packages/cli/package.json

```diff
@@ -7,8 +7,9 @@
   "license": "MIT",
   "scripts": {
     "copy-files": "./scripts/copy-files.sh",
-    "compile-ts": "rm -rf lib && tsc",
-    "prepare": "npm run compile-ts && npm run copy-files && chmod 755 ./lib/bin/oz-cli.js",
+    "compile:ts": "rm -rf lib && tsc",
```

Comment:
Let's keep `compile-ts` to have the same command name in both lib and cli, in order to avoid confusion. We can later add a PR in which we rename all scripts consistently.